cognite/powerops/client/shop/cogshop_api.py

```diff
@@ -18,16 +19,27 @@
 
 
 class CogShopAPI:
-    def __init__(self, cdf: CogniteClient, po: PowerOpsModelsV1Client):
+    def __init__(
+        self,
+        cdf: CogniteClient,
+        po: PowerOpsModelsV1Client,
+        cog_shop_service: Literal["prod", "staging"] | None = None,
+    ):
         self._cdf = cdf
         self._po = po
+        self.cog_shop_service = cog_shop_service
 
     def _shop_url_cshaas(self) -> str:
         project = self._cdf.config.project
 
         cluster = urlparse(self._cdf.config.base_url).netloc.split(".", 1)[0]
 
-        environment = ".staging" if project == "power-ops-staging" else ""
+        if self.cog_shop_service == "prod":
+            environment = ""
+        elif self.cog_shop_service == "staging":
+            environment = ".staging"
+        else:
+            environment = ".staging" if project == "power-ops-staging" else ""

```

Comment:
environment = ".staging" if project == "power-ops-staging" else "" 
is this still needed ?

Reply:
I wanted to leave the current logic as is (if `cog_shop_service` is not specified, then I think it still makes sense to use "staging" for "power-ops-staging" and prod for other projects (?))